content/projects/oop/animals2/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Animals 2!
+ready: true
+---
```

Comment:
We follow git feature branching. One pr should really only cover one thing. They should be specific and small.

The animals2 file has been moved. Please don't add it again to the same place. We'll just have to delete it